judo-tatami-jsl-jsl2psm/src/main/epsilon/operations/jsl/data/errorDeclaration.eol

```diff
@@ -1,26 +1,26 @@
-@cached
-operation JSL!ErrorDeclaration getPSMEquivalent(): JUDOPSM!UnmappedTransferObjectType {
-    return self.equivalent("CreateUnmappedTransferObjectType");
-}
-
-@cached
-operation JSL!ErrorDeclaration getId(): String {
-    return (self.eContainer.getId() + "/" + self.name);
-}
-
-@cached
-operation JSL!ErrorDeclaration getInheritedFields(): Sequence<ErrorField> {
-    var fields = new Sequence<ErrorField>();
-
-    for (field in self.fields) {
-        fields.add(field);
-    }
-
-    if (self.`extends`.isDefined()) {
-        for (f in self.`extends`.getInheritedFields()) {
-            fields.add(f);
-        }
-    }
-
-    return fields;
-}
+//@cached
+//operation JSL!ErrorDeclaration getPSMEquivalent(): JUDOPSM!UnmappedTransferObjectType {
+//    return self.equivalent("CreateUnmappedTransferObjectType");
+//}
+//
+//@cached
+//operation JSL!ErrorDeclaration getId(): String {
+//    return (self.eContainer.getId() + "/" + self.name);
+//}
+//
+//@cached
+//operation JSL!ErrorDeclaration getInheritedFields(): Sequence<ErrorField> {
+//    var fields = new Sequence<ErrorField>();
+//
+//    for (field in self.fields) {
+//        fields.add(field);
+//    }
+//
+//    if (self.`extends`.isDefined()) {
+//        for (f in self.`extends`.getInheritedFields()) {
+//            fields.add(f);
+//        }
+//    }
+//
+//    return fields;
+//}
```

Comment:
Is there any intention to keep this? Any alternative for Error transformation?

Reply:
file removed